plugins/fileservices/fileservices.cpp

```diff
@@ -2644,4 +2648,33 @@ FILESERVICES_API char * FILESERVICES_CALL fsGetEspURL(const char *username, cons
     return strdup("");
 }
 
+FILESERVICES_API char * FILESERVICES_CALL fsGetDefaultDropZone(const char *username, const char *userPW)
+{
+    Owned<IEnvironmentFactory> envFactory = getEnvironmentFactory(true);
+    Owned<IConstEnvironment> constEnv = envFactory-> openEnvironment();
+    Owned<IConstDropZoneInfoIterator> dropZoneIt = constEnv->getDropZoneIterator();
```

Comment:
Is it documented (or JIRA to document) that default dropzone means 1st one listed in environment?

Might be worth making default explicit, e.g. an attribute on the default DropZone, so don't need to reorder.

Reply:
I can't recall it is documented somewhere. @JamesDeFabia do you know?
Ok, but I think it can be another JIRA.

Reply:
I don't think the concept of a default drop zone has ever been mentioned in docs. I agree with Jake that it would be better to explicitly declare one to be the default rather than letting the first one be the default.

Reply:
Agreed as a sensible, but not for this JIRA/PR.